public/index.html

```diff
@@ -28,8 +28,6 @@
     <base href="%REACT_APP_BASE%" />
     <title>Greenhouse Effect</title>
     <script>
```

Comment:
This whole `<script/>` should be removed and auto-size should be used instead. Do you think it would be possible to use the context in this app ? or is the refactoring effort too big ?